MANUAL.md

```diff
@@ -117,6 +117,19 @@ GMake.
 
 There are other functions that do not work in GMake.
 
+#### `$(pname file)` ####
+
+The `pname` function converts a GUARDIAN file name into an OSS path. The
+GUARDIAN name can be fully or partially qualified and does not need to exist.
+The resulting OSS path will always be fully qualified. If the EXPAND node in
+the GUARDIAN name is the same as the local node, the OSS path is `/G` relative
+otherwise the OSS path is `/E` relative.
```

Comment:
I'd mentione that the `pname` function behaves like the `pname -s'  command

It is not relative, asit starts with a /

Does it handled the `/E/node/G/vol/subvol/file` case too? I.e. the middle `/G/`?

Reply:
What's the term, then? Qualified from `/E` or `/G`? Yes, it supports the full `/E/node/G/vol/subvol/file` format. Yes, you're correct, it should mention `pname -s.`

Reply:
'the OSS paths starts with...'

Reply:
Updated the push.